frontend/src/metabase/visualizations/echarts/cartesian/model/axis.ts

```diff
@@ -457,6 +459,26 @@ export function getYAxesModels(
   };
 }
 
+export const getXAxisEChartsType = (
```

Comment:
Moved it to the model since logic becomes more complex and will continue for the further x-axis fixes